pygyro/tools/getPhiSlice.py

```diff
@@ -1,26 +1,29 @@
-from mpi4py                 import MPI
+from .. import splines as spl
+from ..initialisation import constants
+from ..model.process_grid import compute_2d_process_grid
+from ..model.grid import Grid
+from ..model.layout import LayoutSwapper
```

Comment:
Again, strange import order. Doesn't codacy/pylint complain about this?

Reply:
No, but I changed the order.